### PR TITLE
3.1.0-RC1

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**

### CODE REVIEW

File: common/build.gradle [hunk 1]
```diff
@@ -292,14 +292,14 @@ afterEvaluate {
                 groupId 'com.microsoft.identity'
                 artifactId 'common'
                 //Edit the 'version' here for VSTS RC build
-                version = project.version
+                version = '3.0.10-RC2'
```
Comment (AdamBJohnsonx) [inline common/build.gradle:295]:
```suggestion
                version = '3.1.0-RC1'
```

File: common/build.gradle [hunk 2]
```diff
@@ -292,14 +292,14 @@ afterEvaluate {
...
             }
             debug(MavenPublication) {
                 from components.debug
                 groupId 'com.microsoft.identity'
                 artifactId 'common-debug'
                 //Edit the 'version' here for VSTS RC build
-                version = project.version
+                version = '3.0.10-RC2'
```
Comment (AdamBJohnsonx) [inline common/build.gradle:302]:
```suggestion
                version = '3.1.0-RC1'
```

File: common/build.gradle (hunk 1 above)
Comment (iambmelt) [inline common/build.gradle]: Does this become 3.1.0-RC1?

Comment (iambmelt) [reply common/build.gradle]: Resolved with latest changes

File: changelog.txt [hunk 3]
```diff
@@ -1,8 +1,11 @@
 V.Next
 ----------
-- Discontinue using Settings.Secure.ANDROID_ID in telemetry. Instead, generate & cache a random GUID. (#1214)
-- Add refresh_on to access tokens(#1190)
-- Add requested_claims to access tokens, and allow credentials to be filtered by RC (#1187)
+
```
Comment (AdamBJohnsonx) [inline changelog.txt:3]:
```suggestion
- [PATCH] Fix InteractiveRequest Bound Service backcompat (#1215)
```

File: changelog.txt [hunk 4]
```diff
@@ -1,8 +1,11 @@
 V.Next
 ----------
-- Discontinue using Settings.Secure.ANDROID_ID in telemetry. Instead, generate & cache a random GUID. (#1214)
-- Add refresh_on to access tokens(#1190)
-- Add requested_claims to access tokens, and allow credentials to be filtered by RC (#1187)
+- [PATCH] Fix InteractiveRequest Bound Service backcompat (#1215)
```
Comment (AdamBJohnsonx) [inline changelog.txt:3]:
```suggestion
- [PATCH] Fix InteractiveRequest Bound Service backcompat (#1215)

```

**Comment by shahzaibj** [comment]
Is this meant to go into dev?

**Comment by AdamBJohnsonx** [comment]
I mean, eventually it goes into master… and then back to dev.  But it shouldn’t be deleting parts of dev.

Adam.

**Pull request closed without merging**